storage/innobase/xtrabackup/utils/percona-xtrabackup-8.0_builder.sh

```diff
@@ -0,0 +1,587 @@
+        exit 1
+    fi
+    CURPLACE=$(pwd)
+    if [ "x$OS" = "xrpm" ]
+    then
+        add_percona_yum_repo
+        yum -y install git wget 
+        if [[ "${RHEL}" -eq 8 ]]; then         
+            PKGLIST+=" binutils-devel python3-pip python3-setuptools python3-wheel"
+            PKGLIST+=" libcurl-devel cmake libaio-devel zlib-devel libev-devel bison make gcc"
+            PKGLIST+=" rpm-build libgcrypt-devel ncurses-devel readline-devel openssl-devel gcc-c++"
+            PKGLIST+=" install vim-common rpmlint"
+            until yum -y install ${PKGLIST}; do
+                echo "waiting"
+                sleep 1
+            done
+            /usr/bin/pip3 install sphinx
+        else
+            until yum -y install epel-release centos-release-scl; do
+                yum clean all
+                sleep 1
+                echo "waiting"
+            done
+            until yum -y makecache; do
+                yum clean all
+                sleep 1
+                echo "waiting"
+            done
+            PKGLIST+=" devtoolset-7-gcc-c++ devtoolset-7-binutils"
+            PKGLIST+=" devtoolset-7-libasan-devel devtoolset-7-libubsan-devel"
+            PKGLIST+=" devtoolset-7-valgrind devtoolset-7-valgrind-devel"
+            PKGLIST+=" wget libcurl-devel cmake cmake3 make gcc gcc-c++ libev-devel openssl-devel rpm-build"
+            PKGLIST+=" libaio-devel perl-DBD-MySQL vim-common ncurses-devel readline-devel readline"
+            PKGLIST+=" zlib-devel libgcrypt-devel bison"
+            PKGLIST+=" socat numactl"
+            if [[ "${RHEL}" -eq 7 ]]; then
+                PKGLIST+=" numactl-libs perl-Digest-MD5  python3-pip python3-docutils python3-setuptools python3-wheel python3-docutils"
+            elif [[ "${RHEL}" -eq 6 ]]; then
+                PKGLIST+=" python34-pip python34-docutils python34-setuptools python34-docutils"
+            fi
+            until yum -y install ${PKGLIST}; do
+                echo "waiting"
+                sleep 1
+            done
+            # Workaround for Centos > 7 Sphinx
+            if [[ "${RHEL}" -lt 7 ]]; then
+                /usr/bin/pip3 install sphinx
+            fi
+        fi
+    else
+        apt-get -y install lsb-release gnupg2
+
+        if [ "${OS_NAME}" != "focal" ]; then
+            add_percona_apt_repo
+        fi
+        until apt-get update; do
+    	    sleep 1
+            echo "waiting"
+        done
+
+        PKGLIST+=" bison cmake devscripts debconf debhelper automake bison ca-certificates libcurl4-openssl-dev"
+        PKGLIST+=" cmake debhelper libaio-dev libncurses-dev libtool libz-dev"
+        PKGLIST+=" libgcrypt-dev libev-dev lsb-release python-sphinx python-docutils"
+        PKGLIST+=" build-essential rsync libdbd-mysql-perl libnuma1 socat"
+
+        if [ "${OS_NAME}" = "cosmic" ]; then
+            PKGLIST+=" libssl1.0-dev"
+        else
+            PKGLIST+=" libssl-dev"
+        fi
```

Comment:
we can remove this block. cosmic is EOL
just install needed package

Reply:
Done